griptape/drivers/web_search/exa_web_search_driver.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+from attrs import define, field
+
+from griptape.artifacts import JsonArtifact, ListArtifact
+from griptape.drivers import BaseWebSearchDriver
+from griptape.utils import import_optional_dependency
+from griptape.utils.decorators import lazy_property
+
+if TYPE_CHECKING:
+    from exa_py.api import Exa
+
+
+@define
+class ExaWebSearchDriver(BaseWebSearchDriver):
+    api_key: str = field(kw_only=True)
```

Comment:
Do we want to add an Env Var to set this as a second option

Reply:
Looks like [exa-py](https://github.com/exa-labs/exa-py/blob/master/exa_py/api.py#L480) already does this. @william-price01 you can make this field `default=None`